pupa/scrape/schemas/vote.py

```diff
@@ -7,7 +7,7 @@
     "properties": {
         'identifier': {"type": "string", "blank": True,},
         'motion_text': {"type": "string" },
-        'motion_classification': {"items": {"type": "string", "enum": common.VOTE_CLASSIFICATIONS},
+        'motion_classification': {"items": {"type": "string"},
```

Comment:
What if these are settings and if it is blank then no validation is
performed (by dynamically checking for that in the schema nad not adding
the enum key if so)

On Tue, Feb 24, 2015 at 2:50 PM, James McKinney <notifications@github.com>
wrote:

> In pupa/scrape/schemas/vote.py
> <https://github.com/opencivicdata/pupa/pull/149#discussion_r25285718>:
>
> > @@ -7,7 +7,7 @@
> >      "properties": {
> >          'identifier': {"type": "string", "blank": True,},
> >          'motion_text': {"type": "string" },
> > -        'motion_classification': {"items": {"type": "string", "enum": common.VOTE_CLASSIFICATIONS},
> > +        'motion_classification': {"items": {"type": "string"},
>
> The think is, I'm not normalizing motion classification, and I shouldn't
> have to. If I just add the raw Toronto values to
> python-opencivicdata-django, it's going to be a messy list.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencivicdata/pupa/pull/149/files#r25285718>.
>



-- 

James Turk
Labs Director | Sunlight Foundation <http://www.sunlightfoundation.com/>
202-558-8723 | @JamesTurk <https://twitter.com/jamesturk>

[image: Sunlight Foundation] <http://sunlightfoundation.com/>